crates/cairo-lint-core/src/lints/manual/manual_unwrap_or_default.rs

```diff
@@ -0,0 +1,137 @@
+use cairo_lang_semantic::{db::SemanticGroup, Condition, FixedSizeArrayItems};
+use cairo_lang_defs::plugin::PluginDiagnostic;
+use cairo_lang_diagnostics::Severity;
+use cairo_lang_semantic::{Expr, Arenas};
+
+use cairo_lang_syntax::node::{TypedStablePtr, TypedSyntaxNode};
+use num_bigint::BigInt;
+
+pub const MANUAL_UNWRAP_OR_DEFAULT: &str = "This can be done in one call with `.unwrap_or_default()`";
+
+pub const DEFAULT: &str = "\"Felt252Default::default\"";
```

Comment:
this is incorrect, this is the default function for felt252, here you want `Default::default` and when you're checking a function check that it's from the default trait

Reply:
Yes this was because the match case I was using was a felt252 match case.
So depending on the match type, `Default::default` will return the type appended to Default::default as the function text.

E.g:
- `ByteArrayDefault::default` for ByteArrays
- `Felt252Default::default` for Felt252

I've updated it to check for `Default::default` in the function text, as it denotes that the function is return default value.